src/axom/slic/interface/slic_macros.hpp

```diff
@@ -72,6 +72,20 @@
     }                                                               \
   } while(axom::slic::detail::false_value)
 
+/*!
+ * @brief Macro that logs given error message only on root.
+ * \note Must call `axom::slic::initialize(shouldLogRootMessages=true)`
```

Comment:
Consistency: The param here is inconsistent w/ the one in the actual function `axom::slic::initialize()`

Reply:
yea one left over from the many many naming changes i made.